src/_data/sites/ja.yml

```
@@ -0,0 +1,542 @@
+---
+#------------------------------------------------------------------------------
+# English Site Details
+# The primary ESLint site that is hosted at eslint.org
+# Author: Nicholas C. Zakas
+#------------------------------------------------------------------------------
+
+#------------------------------------------------------------------------------
+# Global Settings
+#------------------------------------------------------------------------------
+
+language:
+  code: ja
+  flag: 🇯🇵
+  name: Japanese
+locale: ja-JP
+hostname: ja.eslint.org
+
+#------------------------------------------------------------------------------
+# Analytics
+#------------------------------------------------------------------------------
+
+google_analytics:
+  code: ""
+
+#------------------------------------------------------------------------------
+# Ads
+#------------------------------------------------------------------------------
+
+carbon_ads:
+  serve: ""
+  placement: ""
+
+#------------------------------------------------------------------------------
+# Shared
+#------------------------------------------------------------------------------
+
+shared:
+  get_started: 始める
+  become_a_sponsor: スポンサーになる
+  eslint_logo_alt: ESLintのロゴ
+  description: >
+    JavaScriptのパターンを識別し報告するための、プラグイン可能で設定可能なリンターツールです。
+    コードの品質を簡単に維持することができます。
+  title_format: PAGE_TITLE - ESLint - プラグイン可能なJavaScriptリンターツール
+  skip_to_content: 本文へスキップ
+
+#------------------------------------------------------------------------------
+# Navigation
+#------------------------------------------------------------------------------
+
+navigation:
+  - text: チーム
+    link: team
+  - text: ブログ
+    link: blog
+  - text: ドキュメンテーション
```

```suggestion
  - text: ドキュメント
```
I recommend `ドキュメント` for translation of "Docs" for visual simplicity and generality but not exact.
Compliant to translations of https://github.com/eslint/new.eslint.org/pull/193#discussion_r852602973 also.